### PR TITLE
Refactor/part 10.UI ux enhancement

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/components/remove/remove-liquidity-select-result/RemoveLiquiditySelectResult.tsx [hunk 1]
```diff
@@ -1,64 +1,99 @@
...
+              <MissingLogo
+                symbol={pooledTokenInfo.token.symbol}
+                url={pooledTokenInfo.token.logoURI}
+                width={24}
+                mobileWidth={24}
+              />
+              <p>Unclaimed {pooledTokenInfo.token.symbol}</p>
+              <strong>
+                {removeTrailingZeros(
+                  BigNumber(pooledTokenInfo.amount).toFormat(6),
```
Comment (jinoosss) [inline packages/web/src/components/remove/remove-liquidity-select-result/RemoveLiquiditySelectResult.tsx:74]: Doesn't this require decimals in the token?

File: packages/web/src/components/remove/remove-liquidity-select-result/RemoveLiquiditySelectResult.tsx [hunk 2]
```diff
@@ -1,64 +1,99 @@
...
+              <MissingLogo
+                symbol={pooledTokenInfo.token.symbol}
+                url={pooledTokenInfo.token.logoURI}
+                width={24}
+                mobileWidth={24}
+              />
+              <p>Pooled {pooledTokenInfo.token.symbol}</p>
+              <strong>
+                {removeTrailingZeros(
+                  BigNumber(pooledTokenInfo.amount).toFormat(6),
```
Comment (jinoosss) [inline packages/web/src/components/remove/remove-liquidity-select-result/RemoveLiquiditySelectResult.tsx:55]: Doesn't this require decimals in the token?

File: packages/web/src/constants/common.constant.ts [hunk 3]
```diff
@@ -194,18 +194,22 @@ export const SEOInfo: Record<PageKey, {
...
     },
     desc: () => "Manage your positions to earn trading fees.",
   },
   "token/[token-path]": {
     title: (params = []) => {
-      if (params.length === 3) {
-        const [tokenPrice, tokenName, tokenSymbol] = params;
-        return `${tokenPrice} | ${tokenName}(${tokenSymbol})`;
-      }
+      // Template => `$1.02 | Gnoswap(GNS)`
```
Comment (jinoosss) [inline packages/web/src/constants/common.constant.ts:205]: Remove unused lines.

File: packages/web/src/containers/remove-position-modal-container/RemovePositionModalContainer.tsx [hunk 4]
```diff
@@ -55,53 +105,47 @@ const RemovePositionModalContainer = ({
...
-          maximumFractionDigits: 6,
-        }),
-        tokenBAmount: pooledTokenInfos?.[1]?.amount.toLocaleString("en-US", {
-          maximumFractionDigits: 6,
-        }),
+    const messageData = {
+      tokenASymbol: tokenTransform(pooledTokenInfos?.[0].token).symbol,
+      tokenBSymbol: tokenTransform(pooledTokenInfos?.[1]?.token).symbol,
+      tokenAAmount: pooledTokenInfos?.[0]?.amount.toLocaleString("en-US", {
+        maximumFractionDigits: 6,
```
Comment (jinoosss) [inline packages/web/src/containers/remove-position-modal-container/RemovePositionModalContainer.tsx:119]: Doesn't this require decimals in the token?

File: packages/web/src/containers/remove-position-modal-container/RemovePositionModalContainer.tsx [hunk 5]
```diff
@@ -55,53 +105,47 @@ const RemovePositionModalContainer = ({
...
       }),
-    );
+      tokenBAmount: pooledTokenInfos?.[1]?.amount.toLocaleString("en-US", {
+        maximumFractionDigits: 6,
```
Comment (jinoosss) [inline packages/web/src/containers/remove-position-modal-container/RemovePositionModalContainer.tsx:122]: Doesn't this require decimals in the token?

**Review by jinoosss — COMMENTED** [review]
I left a few minor comments.

**Review by jinoosss — APPROVED** [review]

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**